exercise/exercise_models.py

```diff
@@ -694,7 +695,16 @@ def grade(self, request, submission, no_penalties=False, url_name="exercise"):
         """
         Loads the exercise feedback page.
         """
-        language = get_language()
+        # Get the language from the submission
+        try:
+            aplus_data_index = [i[0] for i in submission.submission_data].index("__aplus__")
+            aplus_data_string = submission.submission_data[aplus_data_index][-1]
+            aplus_data = json.loads(aplus_data_string)
+            language = aplus_data.get("lang",  get_language())
+        except ValueError:
```

Comment:
Does this exception catch all cases: 

* submission has no submission data
* submission data does not contain the `__aplus__` key
* the `__aplus__` object does not contain the `lang` key